Eshop.WebApi/Controllers/CategoryController.cs

```diff
@@ -37,6 +37,7 @@ public Category GetCategory(int id)
         public Category CreateCategory(string name) 
         { 
             var newCategory = new Category(0, name);
+            var newId = dbContext.Categories.Max(x => x.Id) + 1;
```

Comment:
Try to think sometimes, are you using this variable anywhere?
Do you remember its purpose in past?